GitUI/CommandsDialogs/RepoHosting/ViewPullRequestsForm.cs

```diff
@@ -381,9 +391,8 @@ private void SplitAndLoadDiff(string diffData)
                 _diffCache.Add(gis.Name, match.Groups[3].Value);
             }
 
-            // Commits in PR may not exist in the repo so GitRevision in FileStatusList is not used,
-            // patches directly from the cache
-            _fileStatusList.SetDiffs(selectedRev: null, parentRev: null, items: giss);
+            // Note: Commits in PR may not exist in the local repo
+            _fileStatusList.SetDiffs(firstRev: firstRev, secondRev: secondRev, items: giss);
```

Comment:
nit: If arguments are named identically, I would omit the arguement names.
```suggestion
            _fileStatusList.SetDiffs(firstRev, secondRev, items: giss);
```

Reply:
My motivation for keeping the names is that these have changed, but let us assume this is stable now...